x-pack/plugins/siem/public/resolver/view/process_event_dot.tsx

```diff
@@ -485,54 +455,39 @@ const ProcessEventDotComponents = React.memo(
               style={{
                 textTransform: 'uppercase',
                 letterSpacing: '-0.01px',
-                backgroundColor: NamedColors.resolverBackground,
+                backgroundColor: colorMap.resolverBackground,
                 lineHeight: '1',
                 fontWeight: 'bold',
                 fontSize: '0.8rem',
                 width: '100%',
                 margin: '0',
                 textAlign: 'left',
                 padding: '0',
-                color: NamedColors.empty,
+                color: colorMap.descriptionText,
               }}
             >
               {descriptionText}
             </div>
-            <div
-              className={magFactorX >= 2 ? 'euiButton' : 'euiButton euiButton--small'}
-              data-test-subject="nodeLabel"
-              id={labelId}
-              style={{
-                backgroundColor: labelBackground,
-                padding: '.15rem 0',
-                textAlign: 'center',
-                maxWidth: '20rem',
-                minWidth: '12rem',
-                width: '60%',
-                overflow: 'hidden',
-                whiteSpace: 'nowrap',
-                textOverflow: 'ellipsis',
-                contain: 'content',
-                margin: '.25rem 0 .35rem 0',
-              }}
-            >
+            <EuiButton fill={isLabelFilled} color={labelButtonFill} size="s">
```

Comment:
:red_circle: Making this an `<EUIButton>` might cause accessibility problems (and is the reason I chose not to do it before) with the way we've chosen to handle active/focus control for nodes. The reason is that because this is inside a `treeitem` it's supposed to use an "alternate control scheme" according to WCAG 2.0 (i.e. it shouldn't have a tab stop). I think there are some adjustments we _could_ make to do it this way and stay WCAG-compliant, but we'd have to make other adjustments elsewhere to accommodate. Is it / would it be difficult to either: 1) Apply the light theme but keep it as a div as it was before or 2) Somehow keep the EuiButton from creating a tab stop (I _think_ l looked into this before, but maybe worth a doublecheck).

Reply:
Thanks for this. I was wondering about the rationale here, but it makes perfect sense. I'm gonna take a look into the EuiButton, but if it's not possible, I'll go back the div route.

Reply:
So I read up on the `treeItem` spec, and to maintain the functionality of not tabbing over the elements in the tree, I can simply just set `tabIndex=-1` for the EuiButton? The `roving tabIndex` functionality mentioned [here](https://www.w3.org/TR/wai-aria-practices/examples/treeview/treeview-2/treeview-2a.html) and detailed [here](https://www.w3.org/TR/wai-aria-practices/#kbd_roving_tabindex) wasn't how it was implemented before, but I can look into implementing that if that's what you meant? To be true to that pattern we would have to initialize the resolver with one of the nodes automatically focused so I would want to check with @lindseypoli about that

Reply:
Sorry just saw this. 

> I can simply just set tabIndex=-1 for the EuiButton

If you _could_ that would totally work, I think. When I read the docs, it seemed like that would be difficult but it would definitely work. We have an issue up to consider a11y stuff before release (and good point about some of the nodes maybe not even being present @oatkiller ). I'm happy to consider this resolved here, and we can pick it up later.